src/data/support-title.csv

```diff
@@ -69,9 +69,9 @@
 "[おもい、ねがう]","[A Feeling, a Wish]"
 "[Head-on fight！]","[Head-on fight!]"
 "[Two Pieces]","[Two Pieces]"
-"[ツメたいヒトリジメ？]",""
-"[全力！タントラムッッ！]",""
-"[トライフル☆バケーション]",""
+"[ツメたいヒトリジメ？]","[Chilly Hogging?]"
+"[全力！タントラムッッ！]","[Full Power! Tantrum!]"
+"[トライフル☆バケーション]","[Trifle ☆ Vacation]"
```

Comment:
TIL it's a dessert  
https://en.wikipedia.org/wiki/Trifle

Reply:
ikr